packages/rs-drive/src/drive/batch/drive_op_batch/mod.rs

```diff
@@ -345,56 +364,63 @@ mod tests {
             .create_initial_state_structure(Some(&db_transaction))
             .expect("expected to create root tree successfully");
 
-        let contract_cbor = json_document_to_cbor(
+        let contract = json_document_to_contract(
             "tests/supporting_files/contract/dashpay/dashpay-contract-all-mutable.json",
-            Some(crate::drive::defaults::PROTOCOL_VERSION),
         )
-        .expect("expected to get cbor contract");
-        let contract = <Contract as DriveContractExt>::from_cbor(&contract_cbor, None)
-            .expect("contract should be deserialized");
-        let serialized_contract =
-            DriveContractExt::to_cbor(&contract).expect("contract should be serialized");
+        .expect("expected to get contract");
 
-        let _document_type = contract
+        let document_type = contract
             .document_type_for_name("contactRequest")
             .expect("expected to get document type");
 
-        drive_operations.push(ContractOperation(ApplyContractWithSerialization {
+        drive_operations.push(ContractOperation(ApplyContract {
             contract: Cow::Borrowed(&contract),
-            serialized_contract,
             storage_flags: None,
         }));
+        let random_owner_id = rand::thread_rng().gen::<[u8; 32]>();
 
-        let dashpay_cr_serialized_document = json_document_to_cbor(
+        let dashpay_cr_document = json_document_to_document(
             "tests/supporting_files/contract/dashpay/contact-request0.json",
-            Some(1),
+            Some(random_owner_id.into()),
+            document_type,
         )
-        .expect("expected to get cbor contract");
-
-        let random_owner_id = rand::thread_rng().gen::<[u8; 32]>();
+        .expect("expected to get contract");
 
-        drive_operations.push(DocumentOperation(AddSerializedDocumentForContract {
-            serialized_document: dashpay_cr_serialized_document.as_slice(),
-            contract: &contract,
-            document_type_name: "contactRequest",
-            owner_id: Some(random_owner_id),
+        drive_operations.push(DocumentOperation(AddDocumentForContract {
+            document_and_contract_info: DocumentAndContractInfo {
+                owned_document_info: OwnedDocumentInfo {
```

Comment:
I was confused at first but then I've realized that `Owned` means owned by a user, not in a sense of the borrow checker haha